packages/shared/src/components/squads/moderation/SquadModerationList.tsx

```diff
@@ -42,65 +48,97 @@ interface SquadModerationListProps {
   squad: Squad;
 }
 
-export function SquadModerationList({
+const EmptyModerationList = ({
   squad,
-}: SquadModerationListProps): ReactElement {
-  const { onApprove, onReject, isPending } = useSquadPostModeration();
-  const { data, isFetched } = useSquadPendingPosts(squad?.id);
-
-  if (!data?.length) {
-    if (!isFetched || !squad) {
-      return (
-        <div className="flex flex-col gap-4">
-          {placeholder}
-          {placeholder}
-        </div>
-      );
-    }
-
-    if (squad.currentMember.role === SourceMemberRole.Member) {
-      return (
-        <SquadEmptyScreen
-          Icon={VIcon}
-          title="All done!"
-          description="All caught up! No posts are pending" // TODO:: MI-597 - check with product what to show here
-        />
-      );
-    }
+  isFetched,
+}: {
+  squad: Squad;
+  isFetched: boolean;
+}) => {
+  if (!isFetched || !squad) {
+    return (
+      <div className="flex flex-col gap-4">
+        {placeholder}
+        {placeholder}
+      </div>
+    );
+  }
 
+  if (squad.currentMember.role === SourceMemberRole.Member) {
     return (
       <SquadEmptyScreen
         Icon={VIcon}
         title="All done!"
-        description="All caught up! There are no posts waiting for your review right now."
+        description="All caught up! No posts are pending" // TODO:: MI-597 - check with product what to show here
```

Comment:
I think we can just use what we have here. If they have objections, we can always update the copy.

```suggestion
        description="All caught up! No posts are pending"
```

Reply:
Done ✔️